src/ports/postgres/modules/graph/hits.sql_in

```diff
@@ -0,0 +1,297 @@
+        );
+INSERT INTO vertex VALUES
+(0),
+(1),
+(2),
+(3),
+(4),
+(5),
+(6);
+INSERT INTO edge VALUES
+(0, 1, 1),
+(0, 2, 1),
+(0, 4, 1),
+(1, 2, 1),
+(1, 3, 1),
+(2, 3, 1),
+(2, 5, 1),
+(2, 6, 1),
+(3, 0, 1),
+(4, 0, 1),
+(5, 6, 1),
+(6, 3, 1);
+</pre>
+
+-# Compute the HITS scores:
+<pre class="syntax">
+DROP TABLE IF EXISTS hits_out, hits_out_summary;
+SELECT madlib.hits(
+             'vertex',             -- Vertex table
+             'id',                 -- Vertex id column
+             'edge',               -- Edge table
+             'src=src, dest=dest', -- Comma delimited string of edge arguments
+             'hits_out');          -- Output table of HITS
+SELECT * FROM hits_out ORDER BY id;
+</pre>
+<pre class="result">
+ id |      authority       |         hub
+----+----------------------+----------------------
+  0 | 8.43871829094767e-07 |    0.338306115082446
+  1 |    0.158459587238488 |    0.527865350447717
+  2 |      0.4056279696903 |    0.675800764727121
+  3 |    0.721775835522934 | 3.95111934817192e-07
+  4 |    0.158459587238488 | 3.95111934817192e-07
+  5 |    0.316385413093535 |    0.189719957843093
+  6 |    0.405199928761725 |    0.337944978189023
+(7 rows)
+</pre>
+<pre class="syntax">
+SELECT * FROM hits_out_summary;
+</pre>
+<pre class="result">
+ __iterations__
+----------------
+              17
+(1 row)
+</pre>
+-# Running HITS with a max_iter of 3 results in different final values:
+<pre class="syntax">
+DROP TABLE IF EXISTS hits_out, hits_out_summary;
+SELECT madlib.hits(
+             'vertex',             -- Vertex table
+             'id',                 -- Vertex id column
+             'edge',               -- Edge table
+             'src=src, dest=dest', -- Comma delimited string of edge arguments
+             'hits_out',           -- Output table of HITS
+             3);                   -- Max iteration
+SELECT * FROM hits_out ORDER BY id;
+</pre>
+ id |    authority    |        hub
+----+-----------------+--------------------
+  0 | 0.0865332738776 |  0.375721659592658
+  1 | 0.1838832069899 |  0.533118571043637
+  2 |  0.432666369388 |   0.65497424442504
+  3 | 0.7030828502555 | 0.0406185577938009
+  4 | 0.1838832069899 | 0.0406185577938009
+  5 | 0.3028664585716 |  0.182783510072104
+  6 | 0.3893997324492 |  0.330025782074632
+(7 rows)
+</pre>
+<pre class="result">
+ __iterations__
+----------------
+              3
+(1 row)
+</pre>
+
+</pre>
+-# Running HITS with a threshold of 0.01 results in different final values:
+<pre class="syntax">
+DROP TABLE IF EXISTS hits_out, hits_out_summary;
+SELECT madlib.hits(
+             'vertex',             -- Vertex table
+             'id',                 -- Vertex id column
+             'edge',               -- Edge table
+             'src=src, dest=dest', -- Comma delimited string of edge arguments
+             'hits_out',           -- Output table of HITS
+             NULL,
+             0.01);                   -- Max iteration
```

Comment:
`Max iteration`, or `threshold`?
Please have the indentation aligned in the SQL query comments too whenever possible.